amazonlinux/issues.md

```diff
@@ -1 +1,2 @@
-[the Amazon Linux forums](https://forums.aws.amazon.com/forum.jspa?forumID=228)
+-	[amazonlinux/container-images/issues](https://github.com/amazonlinux/container-images/issues)
+-	[the Amazon Linux forums](https://forums.aws.amazon.com/forum.jspa?forumID=228)
```

Comment:
This will render a bit strangely in the resulting `README` and Docker Hub:

 # Quick reference (cont.)
 
-	**Where to file issues**:  

	-	[amazonlinux/container-images/issues](https://github.com/amazonlinux/container-images/issues)
-	[the Amazon Linux forums](https://forums.aws.amazon.com/forum.jspa?forumID=228)
 
-	**Supported architectures**: ([more info](https://github.com/docker-library/official-images#architectures-other-than-amd64))  

---

Our markdown "template" is very naive :disappointed: and so just drops in the sub-template as-is.

Reply:
ok so I should just add them in separate lines without the bullets ?

Reply:
:thinking: A tab at the start of the second line would be enough to get it to fit in.

```
-	[amazonlinux/container-images/issues](https://github.com/amazonlinux/container-images/issues)
	-	[the Amazon Linux forums](https://forums.aws.amazon.com/forum.jspa?forumID=228)
```

Reply:
done.